docs/source/guides/time_series.ipynb

```diff
@@ -0,0 +1,278 @@
+   "metadata": {},
+   "source": [
+    "With these two parameters (`gap` and `window_length`) set, we have defined our feature engineering window. Now, we can move onto defining our feature primitives.\n",
+    "\n",
+    "## Time Series Primitives\n",
+    "\n",
+    "There are three types of primitives we'll focus on for time series problems. One of them will extract features from the time index, and the other two types will extract features from our target column. \n",
+    "\n",
+    "### Datetime Transform Primitives\n",
+    "\n",
+    "We need a way of implicating time in our time series features. Yes, using recent temperatures is incredibly predictive in determining future temperatures, but there is also a whole host of historical data suggesting that the month of the year is a pretty good indicator for the temperature outside. However, if we look at the data, we'll see that, though the day changes, the observations are always taken at the same hour, so the `Hour` primitive will not likely be useful. Of course, in a dataset that is measured at an hourly frequency or one more granular, `Hour` may be incrediby predictive. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "65246092",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "datetime_primitives = [\"Day\", \"Year\", \"Weekday\", \"Month\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95d8c86a",
+   "metadata": {},
+   "source": [
+    "The full list of datetime transform primitives can be seen [here](https://featuretools.alteryx.com/en/latest/api_reference.html#datetime-transform-primitives).\n",
+    "\n",
+    "### Delaying Primitives\n",
+    "\n",
+    "The simplest thing we can do with our target column is to build features that are delayed (or lagging) versions of the target column. We'll make one feature per observation in our feature engineering windows, so we'll range over time from `t - gap - window_length` to `t - gap`. \n",
+    "\n",
+    "For this purpose, we can use our `NumericLag` primitive and create one primitive for each instance in our window. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b9e1fa8f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "delaying_primitives = [NumericLag(periods=i + gap) for i in range(window_length)]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03cd4474",
+   "metadata": {},
+   "source": [
+    "### Rolling Transform Primitives\n",
+    "\n",
+    "Since we have access to the entire feature engineering window, we can aggregate over that window. Featuretools has several rolling primitives with which we can achieve this. Here, we'll use the `RollingMean` primitives `RollingMin`, setting the `gap` and `window_length` accordingly. Here, the gap is incredibly important, because when the gap is zero, it means the current observation's taret value is present in the window, which exposes our target.\n",
```

Comment:
> Here, we'll use the `RollingMean` primitives `RollingMin`,

awkward phrasing

Reply:
changing to "Here, we'll use the `RollingMean` and `RollingMin` primitives"